### PR TITLE
AC-847 Nullable view required in onItemSelected LoginValidatorWatcher

### PR DESCRIPTION
**Pull request by pranoppal**
## Description of what I changed

made argument view nullable in onItemSelected listener
## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-847

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [ ] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.kt [hunk 1]
```diff
@@ -17,6 +17,7 @@ import android.graphics.Color
 import android.text.Editable
 import android.text.Html
 import android.text.TextWatcher
+import android.util.Log
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.kt:20]: remove unused imports

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java [hunk 2]
```diff
@@ -320,7 +320,8 @@ public void showToast(int textId, ToastUtil.ToastType toastType) {
 
     private List<String> getLocationStringList(List<LocationEntity> locationList) {
         List<String> list = new ArrayList<>();
-        list.add(getString(R.string.login_location_select));
+		//If spinner is at start option, append a red * to signify requirement
+		list.add(Html.fromHtml(getString(R.string.login_location_select) + getString(R.string.req_star)).toString());
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java:324]: correct the indentation

**Review by rishabh-997 — CHANGES_REQUESTED** [review]
Check the codestyle of Android studio and align it with the OpenMRS codestyle for future PR's

File: openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.kt (hunk 1 above)
Comment (pranoppal) [reply openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.kt:20]: sure

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java (hunk 2 above)
Comment (pranoppal) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java:324]: okay, I'll import the codestyle of OPENMRS

**Comment by pranoppal** [comment]
@rishabh-997 can you check now?

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java (hunk 2 above)
Comment (pranoppal) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java:324]: hello, the indentation is correct in IDE but after pushing the changes its coming like this, will look in stackoverflow pages for solution

File: .idea/eclipseCodeFormatter.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <option name="projectSpecificProfile">
       <ProjectSpecificProfile>
         <option name="formatter" value="ECLIPSE" />
-        <option name="pathToConfigFileJava" value="$PROJECT_DIR$/.idea/OpenMRSFormatter.xml" />
+        <option name="pathToConfigFileJava" value="$PROJECT_DIR$/../codeformatter.xml" />
```
Comment (rishabh-997) [inline .idea/eclipseCodeFormatter.xml:7]: This change is not necessary... Undo it

File: openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.kt [hunk 4]
```diff
@@ -15,16 +15,10 @@ package org.openmrs.mobile.listeners.watcher
...
 import android.text.Editable
-import android.text.Html
 import android.text.TextWatcher
 import android.view.View
-import android.widget.AdapterView
-import android.widget.Button
-import android.widget.EditText
-import android.widget.Spinner
-import android.widget.TextView
+import android.widget.*
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.kt:20]: Don't use start imports. Not a good practice in open source

**Review by rishabh-997 — CHANGES_REQUESTED** [review]
address these changes. If the indentation is fine in IDE, then leave it

File: openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.kt (hunk 4 above)
Comment (pranoppal) [reply openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.kt:20]: okay thanks, I'll keep this in mind

**Comment by pranoppal** [comment]
@rishabh-997 can you check now? 
Also I have one doubt why codecov/patch diff hit is set to target 13.64%? 
Thanks

**Review by rishabh-997 — APPROVED** [review]
Just squash those commits and I'll merge. Good Work...

**Comment by rishabh-997** [comment]
> why codecov/patch diff hit is set to target 13.64%?

That is something that dev4 or dev5 have set to keep a check on the unit tests as those are very important way to check the credibility of the code.

**Comment by pranoppal** [comment]
> > why codecov/patch diff hit is set to target 13.64%?
> 
> That is something that dev4 or dev5 have set to keep a check on the unit tests as those are very important way to check the credibility of the code.

okay, don't know much about unit testing, but will learn along the way. Thanks

**Comment by pranoppal** [comment]
> Just squash those commits and I'll merge. Good Work...

hey, check now,
Thanks

**Pull request merged**